safehttp/incoming_request_test.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/google/go-safeweb/safehttp/safehttptest"
 )
 
+var nilContext context.Context
```

Comment:
My understanding is you did that for testing only. Just create a local variable and add a comment why you're doing this.

Reply:
I want to ensure the function `SetContext` panics when a nil context is passed but the linter complains if a nil context is passed directly to the function. This is a hack that will stop the linter from complaining hopefully.

Reply:
I get that. Could you make it a *local* variable and add a comment in the code explaining what's happening?